.github/workflows/run.e2e-tests.yml

```diff
@@ -157,7 +159,9 @@ jobs:
       - name: Run Tests
         run: |
           go test -v --timeout 0 --tags="e2e bastion_tunnel" ./test-e2e
-          
+
+      - name: Destroy Infra
+        run: ize down infra
```

Comment:
@kobrikx have you tested it? if I'm not mistaken it should be like `ize down --auto-approve`